hasktorch/src/Torch/Typed/Native.hs

```diff
@@ -2338,8 +2338,16 @@ q_zero_point _input = unsafePerformIO $ (cast1 ATen.q_zero_point_t) _input
 -- combinations :: Tensor dtype shape -> Int -> Bool -> Tensor dtype shape
 -- combinations _input _r _with_replacement = unsafePerformIO $ (cast3 ATen.combinations_tlb) _input _r _with_replacement
 
--- lstm_cell :: Tensor dtype shape -> [Tensor dtype shape] -> Tensor dtype shape -> Tensor dtype shape -> Tensor dtype shape -> Tensor dtype shape -> (Tensor dtype shape,Tensor dtype shape)
--- lstm_cell _input _hx _w_ih _w_hh _b_ih _b_hh = unsafePerformIO $ (cast6 ATen.lstm_cell_tltttt) _input _hx _w_ih _w_hh _b_ih _b_hh
+lstm_cell :: Tensor dtype '[batchSize, inputDim] 
+  -> [Tensor dtype '[batchSize, hiddenSize]]
+  -> Tensor dtype '[4 * hiddenSize, inputDim] 
+  -> Tensor dtype '[4 * hiddenSize, hiddenSize] 
+  -> Tensor dtype '[4 * hiddenSize] 
+  -> Tensor dtype '[4 * hiddenSize]  
+  -> (Tensor dtype '[batchSize, hiddenSize],Tensor dtype '[batchSize, hiddenSize])
+lstm_cell _input _hx _w_ih _w_hh _b_ih _b_hh = unsafePerformIO $ (cast6 ATen.lstm_cell_tltttt) _input _hx _w_ih _w_hh _b_ih _b_hh
+  where
```

Comment:
Thank you for your contribution!
Could you add a test of lstm_cell?

Reply:
What kind of test would you like to see and where would you like to see it?

Reply:
we have used doctests in the past. the whole `Native.hs` module is full of them. feel free to replicate the structure of one of them.
what I'm interested in is a test that the statically calculated types match the runtime shape and dtype of the tensor.